### PR TITLE
Added Gradle Project with August 31 Exercises

### PR DESCRIPTION
**Pull request by horacioAlva**

### CODE REVIEW

File: Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaCurrencyFormatter.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package Hackerrank.August31;
+
+import java.util.*;
```
Comment (luisod78) [inline Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaCurrencyFormatter.java:3]: No wildcard imports please
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

File: Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaIntToString.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package Hackerrank.August31;
+
+import java.util.*;
```
Comment (luisod78) [inline Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaIntToString.java:3]: No wildcard imports please
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

File: Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaStaticInitializerBlock.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package Hackerrank.August31;
+
+import java.util.*;
```
Comment (luisod78) [inline Hackerrank Horacio Alvarado/Hackerrank Excercise(Gradle Project)/src/main/java/Hackerrank/August31/JavaStaticInitializerBlock.java:3]: No wildcard imports please
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by edbermeo — APPROVED** [review]
LGTM!

**Pull request merged**